introduction/research-design-principles.qmd

```diff
@@ -11,48 +11,38 @@ With the *MIDA* framework and the declare, diagnose, redesign algorithm in hand,
 
 This section offers succinct discussions of each principle. We will expand on the implications of these principles for specific design choices throughout the book.
 
-
 :::: {.ddbox}
 ::: {.ddbox-title}
 Design principles
 :::
-
```

Comment:
line breaks before or after ::: should be left as is -- those will not show up as line breaks. This will break the compilation of the book. In general in the markdown code, a line break won't show up as one so would only delete them if there are more than one.

Reply:
I'm sorry but I don't think I understand this comment, what do you mean by "line breaks"?

Reply:
looks good now!